CONTRIBUTING.md

```diff
@@ -212,7 +212,7 @@ If you are using a machine with bare metal machine specs, please use
   - Run `gdb --args python <python file> `
   - You can add breakpoints for future loaded libraries
 - To log the compile time arguments passed with `-D` during the kernel build phase:
-  - Build with debug symbols `make build CONFIG=debug`
+  - Run with Watcher enabled: `TT_METAL_WATCHER=120`
```

Comment:
wonder if this should just be a link to the watcher section.  120 is a somewhat long update interval, but having it listed here means we have multiple places to change the docs to keep consistent.